packages/design-system/src/components/snackbar/constants.js

```diff
@@ -45,11 +45,12 @@ export const SnackbarNotificationThumbnail = PropTypes.shape({
 export const SnackbarNotification = PropTypes.shape({
   message: PropTypes.oneOfType([PropTypes.node, PropTypes.string]),
   actionLabel: PropTypes.oneOfType([PropTypes.node, PropTypes.string]),
-  dismissable: PropTypes.bool,
+  dismissible: PropTypes.bool,
   onAction: PropTypes.func,
   onDismiss: PropTypes.func,
   preventActionDismiss: PropTypes.bool,
   preventAutoDismiss: PropTypes.bool,
   timeout: PropTypes.number,
   thumbnail: SnackbarNotificationThumbnail,
+  key: PropTypes.string,
```

Comment:
this was missing, but used in the component

Reply:
`key` is a special React prop. I don't think that needs to be added to prop types? Or am I missing something?

Reply:
That's true - `key` is a special react prop. It's being used as an id here too: https://github.com/google/web-stories-wp/blob/main/packages/design-system/src/components/snackbar/snackbarContainer.js#L146

Probably best to check for an `id` rather than a `key` since `key` doesn't get propagated as a prop, so the line above is meaningless.